.github/workflows/linting.yml

```diff
@@ -0,0 +1,18 @@
+name: Checks
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+jobs:
+  lint:
+    runs-on: ubuntu-20.04
+    concurrency:
+      group: ${{ github.workflow }}-${{ github.ref }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Run mdl
+        uses: actionshub/markdownlint@main
```

Comment:
Is it a standard practice to point to a branch on github? Are there no releases or such?

Reply:
That actually depends on the org ^^ actionshub is usually directly using their master branch as they ensure that is in a usable state.

Both referencing branches and versions are widespread.

Reply:
>  actionshub is usually directly using their master branch as they ensure that is in a usable state.

But at the same time they seem to be using semver for their tags? So shouldn't we still pin on the major version to avoid breaking CI due to an action change?